e2e/test/TwinE2ETests.cs

```diff
@@ -103,17 +103,11 @@ public async Task Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet_Mq
             await Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet(Client.TransportType.Mqtt_Tcp_Only).ConfigureAwait(false);
         }
 
+        // TODO: #682 - intermittently failing with parsing error.
         [TestMethod]
         public async Task Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet_MqttWs()
         {
-            try
-            {
-                await Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet(Client.TransportType.Mqtt_WebSocket_Only).ConfigureAwait(false);
-            }
-            catch (Newtonsoft.Json.JsonReaderException)
-            {
-                // [Ignore] // TODO: #682 - intermittently failing with JSON parsing error, especially on NET47, NET451.
-            }
+            await Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet(Client.TransportType.Mqtt_WebSocket_Only).ConfigureAwait(false);
```

Comment:
Why are you removing the try catch block ?

Reply:
because we want to enable the test and I want to get our regular logging for it, to look at the error (#682)